docs/readmes/feg/s1ap_federated_test.md

```diff
@@ -197,7 +197,11 @@ vagrant ssh magma_test
 
 # inside vagrant vm
 cd magma/lte/gateway/python/integ_tests
-make integ_test TESTS=federated_tests/s1aptests/test_attach_detach.py
+## Individual test(s):
```

Comment:
Maybe something like `## Individual test(s), e.g.` could be better, to make it clear that `test_attach_detach.py` is just an example.